docs/TIGER_STYLE.md

```diff
@@ -156,6 +156,25 @@ Beyond these rules:
   branches and cases clear. Again, consider whether a single `if` does not also need a matching
   `else` branch, to ensure that the positive and negative spaces are handled or asserted.
 
+* Negations are not easy! In the source code, state invariants positively. When
```

Comment:
```suggestion
* Negations are not easy! Therefore, state invariants positively. When
```

Since the context for most of these is source code.

Reply:
What I want to explain here is that, when `foo and bar` is an invariant, `if (!(foo and bar))` is clearer than `if (!foo or !bar)`. "In the source code" refers to the `foo and bar` substring literally present in the file. 

But it's not like the reader could understand what I mean here, unless the reader is me

Reply:
I'll just more forcefully `. State invariants positively`, as a counter-weight and double-pun for `not easy`.

Reply:
How would one triple-pun this?

Reply:
Attempted Hemingway pun in the commit message: https://github.com/tigerbeetle/tigerbeetle/pull/1188/commits/4321b8ccf5821e0052ce7d69efed0fd769408494

Reply:
I saw it and raised you a Heming'egg: https://github.com/tigerbeetle/tigerbeetle/pull/1188#issuecomment-1715636569